modules/containers-signature-verify-enable.adoc

```diff
@@ -0,0 +1,184 @@
+    {
+      "type": "insecureAcceptAnything"
+    }
+  ],
+  "transports": {
+    "docker": {
+      "registry.access.redhat.com": [
+        {
+          "type": "signedBy",
+          "keyType": "GPGKeys",
+          "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+        }
+      ],
+      "registry.redhat.io": [
+        {
+          "type": "signedBy",
+          "keyType": "GPGKeys",
+          "keyPath": "/etc/pki/rpm-gpg/RPM-GPG-KEY-redhat-release"
+        }
+      ]
+    },
+    "docker-daemon": {
+      "": [
+        {
+          "type": "insecureAcceptAnything"
+        }
+      ]
+    }
+  }
+}
+EOF
+----
+
+.. registry.access.redhat.com.yaml
++
+[source,terminal]
+----
+cat <<EOF > registry.access.redhat.com.yaml
+docker:
+     registry.access.redhat.com:
+         sigstore: https://access.redhat.com/webassets/docker/content/sigstore
+EOF
+----
+
+.. registry.redhat.io.yaml
++
+[source,terminal]
+----
+cat <<EOF > registry.redhat.io.yaml
+docker:
+     registry.redhat.io:
+         sigstore: https://registry.redhat.io/containers/sigstore
+EOF
+----
+
+. Next, set the files with a `base64` encode that will be used for the machine config template.
++
+[source,terminal]
+----
+export ARC_REG=$( cat registry.access.redhat.com.yaml | base64 -w0 )
+export RIO_REG=$( cat registry.redhat.io.yaml | base64 -w0 )
+export POLICY_CONFIG=$( cat policy.json | base64 -w0 )
+----
+
+. Create a machine config file which writes the exported files to disk on the worker nodes.
++
+[source,terminal]
+----
+cat > 51-worker-rh-registry-trust.yaml <<EOF
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: worker
+  name: 51-worker-rh-registry-trust
+spec:
+  config:
+    ignition:
+      config: {}
+      security:
+        tls: {}
+      timeouts: {}
+      version: 2.2.0
+    networkd: {}
+    passwd: {}
+    storage:
+      files:
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,${ARC_REG}
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/containers/registries.d/registry.access.redhat.com.yaml
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,${RIO_REG}
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/containers/registries.d/registry.redhat.io.yaml
+      - contents:
+          source: data:text/plain;charset=utf-8;base64,${POLICY_CONFIG}
+          verification: {}
+        filesystem: root
+        mode: 420
+        path: /etc/containers/policy.json
+  osImageURL: ""
+EOF
+----
+
+. Apply the created machine config for the worker to the cluster.
++
+[source,terminal]
+----
+oc apply -f 51-worker-rh-registry-trust.yaml
+----
+
+. While the master node may not be running the operators or workload pods from the Operator Lifecycle Manager (OLM), it is advised to apply the configuration to the master machine config.
```

Comment:
Do you need all this? _While the master node may not be running the operators or workload pods from the Operator Lifecycle Manager (OLM), it is advised to apply the configuration to the master machine config._ (And the user isn't applying the new machine config in this step, right?)
Can you just go with _Create a machine config file which (does something) on the master nodes._ Keep it parallel with the previous step for the workers (personal preference, here, I think, parallelism).